### PR TITLE
IPC: Catch unavailable service properly

### PR DESCRIPTION
**Pull request by okurz**
Fixes the problem that no real error was shown in the web UI in case of an
IPC, e.g. websockets, not being available.

The error message and backtrace from "confess" call actually end up in the
rendered error page.

Example screenshot:
openqa_ipc_exception

### CODE REVIEW

**Comment by okurz** [comment]
@coolo @aaannz

File: lib/OpenQA/Schema/Result/Workers.pm [hunk 1]
```diff
@@ -127,6 +132,7 @@ sub status {
 sub connected {
     my ($self) = @_;
     my $ipc = OpenQA::IPC->ipc;
+    print "NEXT CALL CRASHES\n\n";
```
Comment (coolo) [inline lib/OpenQA/Schema/Result/Workers.pm]: tsss

File: lib/OpenQA/WebAPI/Controller/Admin/Workers.pm [hunk 2]
```diff
@@ -18,20 +18,32 @@ package OpenQA::WebAPI::Controller::Admin::Workers;
...
     $self->render('admin/workers/index');
 }
 
 sub show {
     my ($self) = @_;
 
-    my $worker = $self->db->resultset('Workers')->find($self->param('worker_id'));
-    $self->stash(worker => $worker);
+    my $w = $self->db->resultset('Workers')->find($self->param('worker_id'));
+    $self->stash(workername => $w->name, worker => _extend_info($w));
```
Comment (coolo) [inline lib/OpenQA/WebAPI/Controller/Admin/Workers.pm]: why is the name not yet another hash key?

Comment (okurz) [inline lib/OpenQA/WebAPI/Controller/Admin/Workers.pm]: because I thought … dunno, does not make sense.

**Pull request merged**